api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -71,10 +71,10 @@ message Bootstrap {
 
     // [#not-implemented-hide:] Hide from docs.
     message DeprecatedV1 {
-      // This is the global :ref:`SDS <arch_overview_dynamic_config_sds>` config
+      // This is the global :ref:`SDS <arch_overview_dynamic_config_eds>` config
```

Comment:
IMO this is still a little confusing. Given that this is hidden from docs, I would just remove the ref links and just say this is for v1 REST configuration and is deprecated?

Reply:
SGTM, done.